serviceWorker.js

```diff
@@ -1,6 +1,6 @@
 importScripts('https://storage.googleapis.com/workbox-cdn/releases/5.1.2/workbox-sw.js');
 
-const appVersion = "v24.9.0";
+const appVersion = "v24.10.0";
```

Comment:
This is meant to be (year, month, version) so we can tie issues to release dates...I know that isn't intuitive. Will you please set this to "v24.11.0"? Thanks!